server/application/terminal.go

```diff
@@ -74,6 +76,9 @@ func (s *terminalHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	userName := sessionmgr.Username(ctx)
+	log.Infof("Serving terminal for userName: %s; podName: %s; container: %s; namespace: %s; cluster: %s", userName, podName, container, namespace, a.ClusterName)
```

Comment:
I wonder if this should use log structured fields instead of a semicolon-delimited list.

Reply:
@crenshaw-dev - yes!!!  structured logs would be ideal - it would be nice to be able to parse these.
I used semi-colon for delimiter for this purpose but if there is an example you can point me at how to better structure this log output it would be preferred!